apis/python/src/tiledbsoma/options/_tiledb_create_options.py

```diff
@@ -7,6 +7,7 @@
     Sequence,
     Tuple,
     Type,
+    TypedDict,
```

Comment:
just a note for myself and anybody else who wonders about this change in the future: `TypedDict` was added to `typing` in py3.8, so we can indeed take it out of `typing_extensions`.